typings/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+    "compilerOptions": {
+        "noImplicitAny": true,
+        "strictNullChecks": false,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "types": [],
+        "noEmit": true,
+        "jsx": "react"
+    },
+    "files": [
+        "index.d.ts",
+        "react-datetime-tests.tsx"
```

Comment:
If moving the test file this reference will have to be updated as well (just a friendly reminder).